hooks/useConversations.tsx

```diff
@@ -15,10 +15,15 @@ const useConversations = () => {
   const { address } = useUserProvider();
   const { conversation } = useParams();
   const conversationRef = useRef(conversation as string);
-  const [streamingTitle, setStreamingTitle] = useState("");
+  const [streamingTitle, setStreamingTitle] = useState({
+    title: "",
+    isTikTokAnalysis: false,
+  });
```

Comment:
This is no longer simply `streamingTitle` right?
- would a new state variable for `isTikTokAnalysis` be a simpler implementation?